MainHomework/MainHomework/ShopCollectionViews/ShopViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ViewController.swift
+//  MainHomework
+//
+//  Created by Наталья Владимировна Пофтальная on 11.11.2023.
+//
+
+import UIKit
+
+final class ShopViewController: UIViewController, UISearchBarDelegate {
+    
+    private var collectionView: UICollectionView!
+    private var layout: UICollectionViewFlowLayout!
+    
+    private lazy var searchBar: UISearchBar = {
+        let searchBar = UISearchBar()
+        searchBar.showsCancelButton = false
+        searchBar.delegate = self
+        searchBar.frame = CGRect(x: 0, y: 70, width: view.frame.size.width, height: 40)
+        searchBar.searchBarStyle = UISearchBar.Style.default
+        searchBar.placeholder = " Search Here....."
+        searchBar.sizeToFit()
+        return searchBar
+    }()
+    
+    private var source: [Source] = [Source(image: "planet1",
```

Comment:
при таком форматировании лучше так
```suggestion
    private var source: [Source] = [
    Source(image: "planet1",
```
чтоб все Source были на одном уровне